utils.py

```diff
@@ -22,6 +22,15 @@
 from ruamel import yaml
 import commonmark
 
+from flask_limiter import Limiter
+from flask_limiter.util import get_remote_address
+
+# Implement the rate limiter
+limiter = Limiter(
+    get_remote_address,
+    storage_uri="memory://",
```

Comment:
I managed to get the limiter working, but for us to make this really work on production we'd have to use a persistent storage solution, like memcached, redis or MongoDB. Now, all of these solutions come at a cost, since as far as I can tell, we'd need to pay a cloud provider for this (e.g; [Memcached Cloud](https://devcenter.heroku.com/articles/memcachedcloud)), with varying costs. @TiBiBa Did you have something in mind when opening this PR about this, do you know a way to fire a memcached server on Heroky without incurring on costs?